docs/api/migration-guides/qiskit-runtime-examples.mdx

```diff
@@ -130,204 +124,153 @@ op = SparsePauliOp.from_list(
 state = QuantumCircuit(2)
 state.x(0)
 state.x(1)
-```
 
-<span id="legacy-opflow"></span>
-##### 1.a. Legacy: Convert problem to opflow
+# Define a local backend
+from qiskit_ibm_runtime.fake_provider import FakeManilaV2
+backend = FakeManilaV2()
 
-`qiskit.opflow` provided its own classes to represent both operators
-and quantum states, so the problem defined above would be wrapped as:
+# Define a real backend
+# from qiskit_ibm_runtime import QiskitRuntimeService
+# service = QiskitRuntimeService()
+# backend = service.least_busy(operational=True, simulator=False)
 
-``` python
-from qiskit.opflow import CircuitStateFn, PauliSumOp
+# Circuits and parameters must obey the Instruction Set Architecture (ISA) of a particular backend.
+from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
+
+pm = generate_preset_pass_manager(backend=backend, optimization_level=1)
+isa_state = pm.run(state)
+isa_observables = [operator.apply_layout(isa_state.layout) for operator in op]

```

Comment:
I suggest not unwrapping the operator to be consistent with the legacy example and return a single estimator output, as explained in 

```suggestion
isa_observable = op.apply_layout(isa_state.layout)

```